packages/cli/src/commands/org/config/push.ts

```diff
@@ -18,15 +19,25 @@ export default class Push extends SnapshotBase {
 
   public static flags = {
     ...SnapshotBase.flags,
+    skipPreview: flags.boolean({
+      char: 'f',
```

Comment:
Why the `f` for `skipPreview` ? 🤔

Reply:
My bad. I initially named the flag `--force` and forgot to rename the char.

Reply:
Does `-s` makes sens?